hwilib/commands.py

```diff
@@ -210,9 +209,12 @@ def process_commands(args):
     getkeypol_parser = subparsers.add_parser('getkeypool', help='Get JSON array of keys that can be imported to Bitcoin Core with importmulti')
     getkeypol_parser.add_argument('--internal', action='store_true', help='Indicates that the keys are change keys')
     getkeypol_parser.add_argument('--keypool', action='store_true', help='Indicates that the keys are to be imported to the keypool')
-    getkeypol_parser.add_argument('path_base', help='The prefix of the derivation path')
-    getkeypol_parser.add_argument('start', type=int, help='The index to start at. The first key will be <path_base>/<start>')
-    getkeypol_parser.add_argument('end', type=int, help='The index to end at. The last key will be <path_base>/<end>')
+    getkeypol_parser.add_argument('path_base', help='The prefix of the derivation path, e.g. m/84h/0h/0h for account 0 native SegWit')
```

Comment:
Can we just call it `key_origin` or something since we're making the split?

Reply:
I didn't want to change too much, and the fingerprint isn't in it... We can probably overhaul the command later, e.g. I'd prefer a one-liner that handles both change and receive and just takes two full descriptors.

Reply:
It's partial key origin info, hence the argument split. Confusing as-is for me.